src/requirements.txt

```diff
@@ -9,7 +8,6 @@ torch
 torchvision
 torchaudio
 intel-npu-acceleration-library
-PytorchWildlife

```

Comment:
I had to remove it because of CI ( i.e. it returns error if I instal it from requirements.txt) . It will be fixed once we got the Conda env